src/basic_grasping_perception.cpp

```diff
@@ -112,11 +114,17 @@ class BasicGraspingPerception
     ROS_DEBUG("Filtered for range, now %d points.", static_cast<int>(cloud_filtered->points.size()));
 
     // Transform to grounded
-    pcl::PointCloud<pcl::PointXYZRGB>::Ptr cloud_transformed(new pcl::PointCloud<pcl::PointXYZRGB>);
-    if (!pcl_ros::transformPointCloud(world_frame_, *cloud_filtered, *cloud_transformed, listener_))
-    {
-      ROS_ERROR("Error transforming to frame %s", world_frame_.c_str());
-      return;
+    pcl::PointCloud<pcl::PointXYZRGB>::Ptr cloud_transformed;
+    if (world_frame_.empty()) {
```

Comment:
please follow our existing ROS-1 style (opening braces should be on their own line)